news/2852.api_change

```diff
@@ -0,0 +1 @@
+Python 3.6 and 3.7 are not supported anymore: Hydra now requires Python 3.8+
```

Comment:
It looks like 3.6 had already been dropped => should we only mention 3.7 here?

Reply:
Updated.